src/main/java/de/craften/ui/swingmaterial/MaterialTextField.java

```diff
@@ -206,12 +272,12 @@ void update() {
             }
             Color targetColor;
             if (target.getText().isEmpty() && target.isFocusOwner()) {
-                targetColor = MaterialColor.CYAN_500;
+                targetColor = accentColor;
             } else {
                 if (target.getText().isEmpty()) {
-                    targetColor = MaterialColor.MIN_BLACK;
+                    targetColor = new Color(target.getForeground().getRGB() & 0x00FFFFFF | HINT_OPACITY_MASK, true);//MaterialColor.MIN_BLACK;
                 } else {
-                    targetColor = MaterialColor.LIGHT_BLACK;
+                    targetColor = accentColor; //MaterialColor.LIGHT_BLACK;
```

Comment:
See the `LoginFrame` example I posted above. Both the floating label and focus line use the accent color when focused. The label remains floating and keeps the accent color when there is content to display.

An alternative could be switching to the hint color if you do not wish the accent color to remain when the target component loses focus.